extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageStreamReader.java

```diff
@@ -133,6 +133,7 @@ public BarrageMessage safelyParseFrom(final StreamReaderOptions options,
 
                         msg.firstSeq = metadata.firstSeq();
                         msg.lastSeq = metadata.lastSeq();
+                        msg.tableSize = metadata.tableSize();
```

Comment:
I noticed we're not setting `step`. Should we be?

Reply:
It feels valuable to set something for step; I'll see if I can easily add this in for BMPs.

Reply:
Actually, this step is only used to communicate from ConstructSnapshot which cycle the snapshot belongs to. This is also communicate via firstSeq. I'm going to remove step from the type. In particular, we don't have a field to send this in barrage update metadata.